index.html

```diff
@@ -24,125 +41,629 @@
+    &lt;/ul&gt;
+  &lt;/div&gt;
+&lt;/div&gt;</code></pre>
+        </section>
+        <section id="popover-shadow-invoked" class="demo-item">
+          <h2 data-header>
+            <a href="#popover-shadow-invoked" aria-hidden="true">🔗</a>
+            Opening a Popover with an Element Created in the Shadow Root
+          </h2>
+          <p class="note">
+            Elements created in the shadow DOM can trigger a popover.
+          </p>
+          <button
+            popovertarget="shadowInvokedPopover"
+            data-btn
+            data-popover-control
+          >
+            <span id="shadowInvokedHost"></span>
+          </button>
+          <script type="module">
+            const shadowInvokedHost =
+              document.getElementById('shadowInvokedHost');
+            const shadowRoot = shadowInvokedHost.attachShadow({ mode: 'open' });
+            shadowRoot.innerHTML = `<span>Toggle Shadow Invoked Popover</span>`;
+          </script>
+          <pre><code class="language-html"
+>&lt;!-- Light DOM --&gt;
+&lt;button popovertarget="shadowInvokedPopover"&gt;
+  &lt;span id="shadowInvokedHost"&gt;
+    &lt;!-- Shadow DOM --&gt;
+    &lt;span&gt;Toggle Shadow Invoked Popover&lt;/span&gt;
+  &lt;/span&gt;
+&lt;/button&gt;
+
+&lt;div id="shadowInvokedPopover" popover&gt;
+  Shadow Invoked Popover
+&lt;/div&gt;</code>
+
+<code class="language-js"
+>// JavaScript
+const shadowInvokedHost = document.getElementById('shadowInvokedHost');
+const shadowRoot = shadowInvokedHost.attachShadow({ mode: 'open' });
+shadowRoot.innerHTML = `&lt;span&gt;Toggle Shadow Invoked Popover&lt;/span&gt;`;</code></pre>
+        </section>
+        <section id="popover-shadow-nested" class="demo-item">
+          <h2 data-header>
+            <a href="#popover-shadow-nested" aria-hidden="true">🔗</a>
+            Nested Popovers in Shadow DOM
+          </h2>
+          <p class="note">
+            Popovers can be nested within other popovers in the shadow DOM.
+          </p>
+          <button
+            popovertarget="shadowNestedPopover"
+            data-btn
+            data-popover-control
+          >
+            Toggle Menu with Shadowed Popover
+          </button>
+          <script type="module">
+            const menuHost = document.getElementById('menuHost');
+            const menuShadowRoot = menuHost.attachShadow({ mode: 'open' });
+            menuShadowRoot.innerHTML = `
+              <style>
+                @layer reset, initial, popover-polyfill, default, demo;
+
+                @import url('/css/reset.css') layer(reset);
+                @import url('/css/demo.css') layer(demo);
+                @import url('/css/prism.css') layer(initial);
+                @import url('/css/fonts.css') layer(initial);
+                @import url('/css/base.css') layer(default);
+                @import url('/css/props.css') layer(initial);
+                @import url('/css/code.css') layer(default);
+              </style>
```

Comment:
I'm guessing most of these styles aren't needed inside the shadow root -- should we pare this down?

Reply:
I cleaned this up so that we're only using the bare minimum for the popovers to look like they should in https://github.com/oddbird/popover-polyfill/pull/187/commits/67ee7c4c346c5a4e86b2193dd6c17a7a3cafa3c6